Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_050.IdentifiableNpmAuthorTokenValidator.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.RE2.Managed;
 using Microsoft.Security.Utilities;
 
-namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.SecurePlaintextSecretsValidators
```

Comment:
>SecurePlaintextSecretsValidators [](http://example.com/codeflow?start=71&length=32)

please revert these changes, just use the base namespace.

an overly granular namespace organization just complicates code and lengthens lines of source.

Reply:
just updated!